modules/core/zio/src/weaver/ziocompat/suites.scala

```diff
@@ -49,9 +49,10 @@ abstract class BaseMutableZIOSuite[Res <: Has[_]](implicit tag: Tag[Res])
       else {
         for {
           ref <- Stream.eval(FiberRef.make(Chain.empty[Log.Entry]))
-          testLayer: RLayer[ZEnv, LogModule with ZEnv] =
-            ZEnv.any ++ ZLayer.fromService[Clock.Service, LogModule.Service](
-              FiberRefLog(ref, _))
+          testLayer: RLayer[ZEnv, Live with LogModule with ZEnv] =
+            ZEnv.any >+> Live() ++ ZLayer.fromService[
```

Comment:
So basically if the user override any of the default `ZEnv` services in their sharedLayer, `Live` gives them a way to retain access to the `real` one, and if they don't, they just get the real layers ? Is my understanding correct ?